oidc_apis/models.py

```diff
@@ -186,3 +188,15 @@ class Meta:
 
     def __str__(self):
         return "{obj}[{lang}]".format(obj=self.master, lang=self.language_code)
+
+
+def get_all_required_scopes_by_api_scopes(scopes):
+    api_scopes = ApiScope.objects.by_identifiers(scopes)
+    apis = {x.api for x in api_scopes}
+    return set(sum((list(api.required_scopes) for api in apis), []))
+
+
+def extend_scope_by_api_scopes(scopes):
```

Comment:
How about moving this to a class method such as `ApiScope.extend_scope` (shorter method name should be enough, since the ApiScope class in the name should make it clear that it's somehow related to ApiScopes anyway)